src/xrpld/app/tx/detail/Payment.cpp

```diff
@@ -285,6 +285,13 @@ Payment::checkPermission(ReadView const& view, STTx const& tx)
         return tecNO_DELEGATE_PERMISSION;
     }
 
+    // Pre-fixDelegateV1_1: MPT is not supported here.
+    // Return tefEXCEPTION explicitly rather than relying on dstAmount.issue()
+    // to throw. (This code is only reachable when fixDelegateV1_1 is not
+    // enabled.)
+    if (dstAmount.holds<MPTIssue>())
+        return tefEXCEPTION;
```

Comment:
Please add a comment explaining why this is not a transaction-breaking change requiring an amendment.
* The next line calls `dstAmount.issue()`
* That throws if it holds MPT
* The thrown exception is caught in `preclaim` in `applySteps.cpp`.
* Catching an exception returns `tefEXCEPTION`.
* Therefore, this is just a cleaner, more explicit way to get the same result.